traitquestdev/css/style.css

```diff
@@ -51,8 +62,26 @@ textarea{resize:none;}
 .margin-topbottom-s{margin-top: 16px; margin-bottom: 16px;}
 .margin-topbottom-xs{margin-top: 12px; margin-bottom: 12px;}
 .fontsize-s{font-size:14px;}
+.fontsize-m{font-size:18px;}
+.fontsize-l{font-size:24px;}
+.no-list-style{list-style-type:none;}
+.display-block{display:block;}
+.display-inlineblock{display:inline-block;}
 /*#wrapper{ padding-bottom: 80px;  min-height:100%; position:relative;}*/
 
 /* 3.0 HEADER
 ------------------------------------------ */
-#header{background:#C5EFF7;}
\ No newline at end of file
+#header{background:#C5EFF7;}
```

Comment:
the header background color should be different compared to the one i designed in the dropbox?